### PR TITLE
Added WD column

### PR DESCRIPTION
**Pull request by salgo60**
also removed duplicates

### CODE REVIEW

File: corpus/metadata/government.csv [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
-1991-10-04,1994-10-07,Regeringen Carl Bildt
-1994-10-07,1996-03-22,Regeringen Carlsson III
-1996-03-22,2006-10-06,Regeringen Persson
-2006-10-06,2014-10-03,Regeringen Reinfeldt
-2014-10-03,2019-01-19,Regeringen Löfven I
-2019-01-21,2021-07-09,Regeringen Löfven II
-2021-07-09,2021-11-30,Regeringen Löfven III
-2021-11-30,,Regeringen Andersson
\ No newline at end of file
+start,end,government,wikidata
```
Comment (MansMeg) [inline corpus/metadata/government.csv]: I'm thinking we should try to coordinate the wikidata_id column @ninpnin do you know what we use in other places? I think it is wikidata_id? That make more sense to me?

**Comment by MansMeg** [comment]
Great! I just want to have @ninpnin s comment on what we should call the column. I think it should be wikidata_id, but I'm not sure.

**Comment by ninpnin** [comment]
In other files, we seem to be using ```wiki_id``` , see https://github.com/welfare-state-analytics/riksdagen-corpus/blob/main/corpus/metadata/name.csv#L1

**Comment by MansMeg** [comment]
@salgo60 Do you think wiki_id is good or is wikifata_id more correct? I think we should choose the name that is the most correct. It will be harder to change down the line.

**Comment by salgo60** [comment]
just **Wikidata** I think is good enough see how [Nobelprize.org do it](https://github.com/salgo60/NOSAD-POC-Wikidata/issues/13)

**Comment by MansMeg** [comment]
I think we have the semantic **_id for different ids I think. Then I think we should go with wikifata_id here. @ninpnin  you think this is good? Then Ill also open an issue to renane the other volumns.

**Comment by salgo60** [comment]
Maybe its better having it with ID another approach is like Wikidata has Qnumbers and Pnumbers see [Wikidata:Identifiers](https://www.wikidata.org/wiki/Wikidata:Identifiers)

* the pattern Wikidata has is using numbers e.g. [Property:P8433](https://www.wikidata.org/wiki/Property:P8433) and then have labels in different languages
   * as I guess you are part of [ParlaMint](https://www.clarin.eu/parlamint) I guess you will have people speaking different languages and then a number is maybe easier I guess
 
 Swedish PM Q272930 - 80 languages we have in WD - [SPARQL](https://w.wiki/5mvC)
 
 image   
 
 * What Wikidata is called in 184 languages - [SPARQL](https://w.wiki/5mvd)
 
 image 
 
 Same for [property P8388](https://www.wikidata.org/wiki/Property:P8388) 10 languages - [SPARQL ](https://w.wiki/5mvQ)
 
image

FYI I asked in the Telegram group Wikidata for a second opinion. If you have Telegram account let me know if I should add you.... [Lucas is e.g. senior Wikidata dev](https://phabricator.wikimedia.org/p/Lucas_Werkmeister_WMDE/)

image

**Comment by salgo60** [comment]
@MansMeg 
* wikidata_item_id
* wikidata_entity_id

image

My vision with your work is that we shall also in TEI have same as [Wikidata Lexemes ](https://www.wikidata.org/wiki/Wikidata:Lexicographical_data)--> identifiers are starting with L see  example [Lexeme L54865 - foliehat](https://www.wikidata.org/wiki/Lexeme:L54865#P5831)

image

**Comment by MansMeg** [comment]
I think we only hav wikidata Q ids? Is that an entity or an item? :) im happy with either, but if you come from wikidata and see a Q number what alt do you think would be the best one?

**Comment by salgo60** [comment]
@MansMeg then I need to look in the books :smiley: I am just a Wikidata Volontaire... Lucas [WD Q57387675](https://www.wikidata.org/wiki/Q57387675) is 100 times more skilled than me his Master Thesis [WD Q57385555 ](https://www.wikidata.org/wiki/Q57385555) "[Schema Inference on Wikidata](https://github.com/lucaswerkmeister/master-thesis/releases/tag/final)"....

I found the ontology [DINGO](https://dcodings.github.io/DINGO/) they use [dg:wikidata_item_id](https://dcodings.github.io/DINGO/#wikidata_item_id)

image

**OT 11:45 today you have how nice Nobelprize.org** use Linked data and same as Wikidata see https://github.com/salgo60/NOSAD-POC-Wikidata/issues/13#issuecomment-1267925234

**Comment by MansMeg** [comment]
Ok. Then I suggest we go with wikidata_item_id. We can change later on, but if others use this it is probably the best.

could you change this in the PR @salgo60 ? Then Ill file an issue to change this throughout.

**Comment by salgo60** [comment]
Done https://github.com/welfare-state-analytics/riksdagen-corpus/pull/199/files

**Comment by salgo60** [comment]
* WD [Q114671310](https://www.wikidata.org/wiki/Q114671310) = "[Regeringen Kristersson](https://sv.wikipedia.org/wiki/Regeringen_Kristersson)" - [graf](https://w.wiki/5q5t)

[image](https://w.wiki/5q5t)

**Comment by MansMeg** [comment]
Great! Could you do a new pull request with this?

**Comment by salgo60** [comment]
@MansMeg yes I can do

We get a lot of new public offices etc... when this gets stable.... do you have written down what objects you have and synch with Wikidata?

* Public Offices [list](https://w.wiki/5q94)
image
image

**Comment by MansMeg** [comment]
Yes. There is no rush with this, so let it calm down a little. =)

**Pull request merged**